tests/results/ffa6e3647b8f4d69fec285f355931a20c76b9f2a7b106a298ec460ec572c8ef8_empty_name/result.json

```diff
@@ -343,6 +339,10 @@
         "name": "msmm.png",
         "sha256": "1ab5ef4e7e196cb1ff39df44e1a0a39f6880b906ef6fd6da3cfdbb92ffd33115"
       },
+      {
+        "name": "jupiter.js",
```

Comment:
Why was this file transferred from extracted to supplementary? This is one of the two tests that are failing on the buildhost.